lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/search/query/Suggestion.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description":"JSON schema for a suggestion.",
```

Comment:
How about "SuggestionList", and
description: "JSON Schema for a list of suggested replacements for a search term"
?